### ISSUE TITLE
Typo in 3.2.4 Sufficient Technique Note 1

### ISSUE THREAD

**Issue by yatil** [w3c/wcag#925]
See this report on the quickref: https://github.com/w3c/wai-wcag-quickref/issues/198

Under https://www.w3.org/WAI/WCAG21/Understanding/consistent-identification.html#sufficient

> Text alternatives that are "consistent" are not always "identical." For instance, you may have **an graphical arrow** at the bottom of a Web page that links to the next Web page. The text alternative may say "Go to page 4." Naturally, it would not be appropriate to repeat this exact text alternative on the next Web page. It would be more appropriate to say "Go to page 5". Although these text alternatives would not be identical, they would be consistent, and therefore would satisfy this Success Criterion.

should be

> Text alternatives that are "consistent" are not always "identical." For instance, you may have **a graphical arrow** at the bottom of a Web page that links to the next Web page. The text alternative may say "Go to page 4." Naturally, it would not be appropriate to repeat this exact text alternative on the next Web page. It would be more appropriate to say "Go to page 5". Although these text alternatives would not be identical, they would be consistent, and therefore would satisfy this Success Criterion.

**Comment by awkawk** [comment]
Fixed in Master. Thanks.
